### PR TITLE
README tweaks

### PR DESCRIPTION
**Pull request by pelotom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,27 +58,28 @@ const todosReducer = (state: Todo[] = [], action: Action) =>
...
 
- `action` can be omitted. In that case the result of match is a function
+`action` can be omitted; in that case the result of match is a function:
 
 ```ts
 const getIdFromAction = Action.match({
   ADD_TODO: ({ id}) => id,
   TOGGLE_TODO: ({ id }) => id,
-  default: a => `${a.tag} doesn't have id`
+  default: a => undefined,
```
Comment (pelotom) [inline README.md]: Should this throw an exception referencing `a.type`? It seems very odd to return a `string` which could be confused for an ID.

Comment (twop) [reply README.md]: id is a string type. So i guess throwing an exception would be a better example

Comment (twop) [reply README.md]: Yes I like it more 👍

File: README.md [hunk 2]
```diff
@@ -140,8 +147,8 @@ That allowed to introduce inline matching
 const light = Light.Off({});
 
 //before you had to pass an object using ().
-Light.match({On: ()=> 'is on'}, ()=>'is off')(light);
+Light.match({On: () => 'is on'}, () =>'is off')(light);
 
 //now
-Light.match(light, {On: ()=> 'is on', default: ()=>'is off'})
+Light.match(light, {On: () => 'is on', default: () =>'is off'})
```
Comment (twop) [inline README.md]: I think it makes sense to demonstrate the point
```ts
// now you can do both
Light.match(light, {On: () => 'is on', default: () =>'is off'});
Light.match({On: () => 'is on', default: () =>'is off'})(light);
```

File: README.md [hunk 3]
```diff
@@ -116,32 +123,37 @@ Overall, it is pretty similar to `match` but reduces boilerplate. Also note that
 #### Breaking changes from 1.1
```
Comment (twop) [inline README.md]: My bad. Should be 1.0.1

Comment (pelotom) [reply README.md]: Good catch.

File: README.md [hunk 4]
```diff
@@ -113,35 +120,40 @@ const toggled = Light.transform(on, {
...
-//before
+// before
 unionize({...}, 'myTag', 'myPayloadProp');
+unionize({...}, 'myTag');
 
-//now
-unionize({...}, {tag:'myTag', value:'myPayloadProp'});
+// after
+unionize({...}, { tag:'myTag', value:'myPayloadProp' });
+unionize({...}, { tag:'myTag', value:'myPayloadProp' });
```
Comment (twop) [inline README.md]: duplicate line?

**Comment by twop** [comment]
Looks awesome! 👍

**Pull request merged**